ocs_ci/ocs/ui/views.py

```
@@ -1085,6 +1085,20 @@
 }
 
+acm_page_nav_422 = {
+    # PF6: button text lives in a child span, so match on the span text
+    "Infrastructure": (
+        "//button[normalize-space()='Infrastructure'"
+        " or .//span[normalize-space()='Infrastructure']]",
+        By.XPATH,
+    ),
+    # PF6: cluster status is a button whose text is inside pf-v6-c-button__text span
+    "cluster_status_check": (
+        "//button[normalize-space()='{}'" " or .//span[normalize-space()='{}']]",
```

Ruff (0.15.17)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)